Change the description of shell profile and add parameter -i in quick-start-with-tidb.md (#8293)
pingcap/docs: quick-start-with-tidb.md

    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
    ```

    Verify that TiUP has installed successfully by checking the output in the terminal of shell window for the following message:

    ```log
    Successfully set mirror to https://tiup-mirrors.pingcap.com
    Detected shell: zsh
    Shell profile:  /Users/user/.zshrc
    /Users/user/.zshrc has been modified to add tiup to PATH
    open a new terminal or source /Users/user/.zshrc to use it
    Installed path: /Users/user/.tiup/bin/tiup
    ===============================================
    Have a try:     tiup playground
    ===============================================
    ```

> 

Reviewer: We should tell the user to copy or at least remember the shell profile path here.

2. Declare the global environment variable:

    > **Note:**
    >
    > After the installation, TiUP displays the absolute path of the corresponding Shell profile file. You need to modify `${your_shell_profile}` in the following `source` command according to the path.

    {{< copyable "shell-regular" >}}

    ```shell

>     source ${your_shell_profile}

Reviewer: Can we add a sentence like:
```
In this case, `${your_shell_profile}` is `/Users/user/.zshrc` from the output of Step 1.
```

    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
    ```

>     Verify that TiUP has installed successfully by checking the output in the terminal or shell window for the following message:

Reviewer 2: ```suggestion
    If the following message is displayed, you have installed TiUP successfully:
```

    ```log
    Successfully set mirror to https://tiup-mirrors.pingcap.com
    Detected shell: zsh
    Shell profile:  /Users/user/.zshrc
    /Users/user/.zshrc has been modified to add tiup to PATH
    open a new terminal or source /Users/user/.zshrc to use it
    Installed path: /Users/user/.tiup/bin/tiup
    ===============================================
    Have a try:     tiup playground
    ===============================================
    ```

>     Please pay attention to the Shell profile path above. After installation, this file has been modified to add tiup to PATH. You can open a new terminal or follow the next step to declare the global environment variables of the TiUP.

Reviewer: 1. Let's not overdo the politeness. ([ref](https://developers.google.com/style/tone))
2. "You can open a new terminal or follow the next step" might confuse the user.

```suggestion
    Note the Shell profile path in the output above. You need to use the path in the next step.
```

    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
    ```

>     Verify that TiUP has installed successfully by checking the output in the terminal of shell window for the following message:

Reviewer 2: ```suggestion
    If the following message is displayed, you have installed TiUP successfully:
```
